develop/tutorials/articles/28-themes-and-layout-templates/01-application-decorators/00-app-decor-intro.markdown

```diff
@@ -0,0 +1,9 @@
+# Application Decorators [](id=application-decorators)
+
+Would you like to customize your application's look and feel? Create Application
+Decorators! Application Decorators allow you to create unique styles for your
+application's wrapper. Once your Application Decorators are created, an admin 
+can choose a decorator for each application.
+
```

Comment:
This paragraph doesn't tell me anything about what Application Decorators are, but the first paragraph of the intro file below does. I recommend you replace the content in this file with what's in that one.